scripts/split-paired-reads.py

```diff
@@ -92,11 +92,13 @@ def main():
 
     infile = args.infile
 
-    check_input_files(infile, args.force)
     filenames = [infile]
     check_space(filenames, args.force)
-
     # decide where to put output files - specific directory? or just default?
+    if os.path.basename(infile) == 'stdin':
```

Comment:
I think this should just be 'if infile == '/dev/stdin' - no need to use basename.  Also, check for infile == '-'.